fcrepo-import/src/main/java/org/fcrepo/importer/Importer.java

```diff
@@ -17,26 +17,178 @@
  */
 package org.fcrepo.importer;
 
+import static org.apache.jena.rdf.model.ModelFactory.createDefaultModel;
+import static org.apache.jena.rdf.model.ResourceFactory.createResource;
+import static org.slf4j.LoggerFactory.getLogger;
+import static org.fcrepo.importexport.FcrepoConstants.CONTAINS;
+import static org.fcrepo.importexport.FcrepoConstants.DESCRIBEDBY;
+import static org.fcrepo.importexport.FcrepoConstants.HAS_MIME_TYPE;
+import static org.fcrepo.importexport.FcrepoConstants.HAS_MESSAGE_DIGEST;
+import static org.fcrepo.importexport.FcrepoConstants.HAS_SIZE;
+import static org.fcrepo.importexport.FcrepoConstants.NON_RDF_SOURCE;
+import static org.fcrepo.importexport.FcrepoConstants.RDF_TYPE;
+import static org.fcrepo.importexport.FcrepoConstants.REPOSITORY_NAMESPACE;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.commons.io.IOUtils;
+import org.apache.jena.rdf.model.Model;
+import org.apache.jena.rdf.model.ResIterator;
+import org.apache.jena.rdf.model.Statement;
+import org.apache.jena.rdf.model.StmtIterator;
+import org.fcrepo.client.FcrepoClient;
+import org.fcrepo.client.FcrepoOperationFailedException;
+import org.fcrepo.client.FcrepoResponse;
 import org.fcrepo.importexport.TransferProcess;
+import org.slf4j.Logger;
 
 /**
+ * Fedora Import Utility
+ *
  * @author awoods
+ * @author escowles
  * @since 2016-08-29
  */
 public class Importer implements TransferProcess {
+    private static final Logger logger = getLogger(Importer.class);
+    private Config config;
+    protected FcrepoClient client;
 
     /**
      * Constructor that takes the Import/Export configuration
      *
      * @param config for import
      */
     public Importer(final Config config) {
+        this.config = config;
+        this.client = FcrepoClient.client().build();
```

Comment:
I kind of like the idea of passing in the `client`, especially if we will be allowing for authorized clients.